src/utils/session-helper.js

```diff
@@ -249,3 +251,36 @@ export const updateTTLFile = async (session, containerUrl, fileObject) => {
     throw new Error('Failed to update ttl file.');
   }
 };
+
+
+
+/**
+ * Creates a TTL file corresponding to an uploaded document or resource
+ *
+ * @memberof utils
+ * @function createResourceTtlFile
+ * @param {fileObjectType} fileObject - Object containing information about file
+ * from form submission (see {@link fileObjectType})
+ * @param {documentUrl} String - url of uploaded document or resource
+ * @returns {object} 
+ */
+export const createResourceTtlFile = async (fileObject, documentUrl) => {
+  let checksum = await createFileChecksum(fileObject);
+
+  return buildThing(createThing({ name: 'document' }))
+    .addDatetime(RDF_PREDICATES.uploadDate, new Date())
+    .addStringNoLocale(RDF_PREDICATES.name, fileObject.file.name)
+    .addStringNoLocale(RDF_PREDICATES.identifier, fileObject.type)
+    .addStringNoLocale(RDF_PREDICATES.endDate, fileObject.date)
+    .addStringNoLocale(RDF_PREDICATES.serialNumber, checksum)
+    .addStringNoLocale(RDF_PREDICATES.description, fileObject.description)
+    .addUrl(RDF_PREDICATES.url, documentUrl)
+    .build();
+}
+
+const createFileChecksum = async (fileObject) => {
+  const file = fileObject.file;
```

Comment:
On Line 282, since we're just calling file from fileObject, we can use object destructuring:

```JavaScript
const { file } = fileObject;
```

This should have the same effect as:

```JavaScript
const file = fileObject.file;
```

Reply:
fixed